src/main/java/com/inscopemetrics/mad/parsers/StatsdToRecordParser.java

```diff
@@ -227,7 +239,7 @@ private StatsdToRecordParser(final Builder builder) {
     private static final Splitter LINE_SPLITTER = Splitter.on('\n').omitEmptyStrings();
     private static final ThreadLocal<NumberFormat> NUMBER_FORMAT = ThreadLocal.withInitial(NumberFormat::getInstance);
     private static final Pattern STATSD_PATTERN = Pattern.compile(
-            "^(?<NAME>[^:@|]+):(?<VALUE>[^|]+)\\|(?<TYPE>[^|]+)(\\|@(?<SAMPLERATE>[^|]+))?(\\|#(?<TAGS>.+))?$");
+            "^(?<NAME>[^:@|,]+)(,(?<INFLUXTAGS>[^:@|]+))?:(?<VALUE>[^|]+)\\|(?<TYPE>[^|]+)(\\|@(?<SAMPLERATE>[^|]+))?(\\|#(?<TAGS>.+))?$");
```

Comment:
What statsd protocol documentation supports this? Can you update the documentation for the class with appropriate descriptions and references?

Reply:
Updated documentation in 63d1d0829d804b44a945dfde1f1129033f784127